ui/ui-enterprise/__tests__/integration/happyPath.ts

```diff
@@ -90,29 +77,38 @@ describe("integration", () => {
     "I was taken to the setup page after creating the first team",
     async () => {
       await browser.wait(until.elementLocated(by.id("setup")));
-      const url = await browser.getCurrentUrl();
-      expect(url).toMatch(/\/setup/);
-      await browser.get(url);
+      const currentURL = await browser.getCurrentUrl();
+      expect(currentURL).toEqual(`${url}/setup`);
     },
     helper.mediumTime
   );
 
+  test("The new user was saved to the database and session created", async () => {
+    const teamMembers = await TeamMember.findAll();
+    expect(teamMembers.length).toBe(1);
+    expect(teamMembers[0].email).toBe(email);
+
+    const sessions = await Session.findAll();
+    expect(sessions.length).toBe(1);
+    expect(sessions[0].teamMemberId).toBe(teamMembers[0].id);
+  });
+
   test(
     "I can sign out",
     async () => {
-      await browser.get(`${env.url}/session/sign-out`);
+      await browser.get(`${url}/session/sign-out`);
       await browser.wait(until.elementLocated(by.tagName("p")));
-      const url = await browser.getCurrentUrl();
-      expect(url).toMatch(/\//);
-      await browser.get(url);
```

Comment:
I see that this line is no longer used in the tests... what was its purpose before we switched the specHelper?  Why is it no longer necessary?  I was curious why we would `browser.get(url)` both before and after the test.

Reply:
I also don't know!  

I agree that it seems unnecessary, since we already confirmed the page was `/`... and the test works without those lines.